include/dem/force_chains_visualization.h

```diff
@@ -0,0 +1,61 @@
+#include <core/auxiliary_math_functions.h>
+#include <core/dem_properties.h>
+
+#include <dem/data_containers.h>
+#include <dem/dem_contact_manager.h>
+#include <dem/dem_solver_parameters.h>
+#include <dem/particle_particle_contact_force.h>
+#include <dem/particle_particle_contact_info.h>
+#include <dem/rolling_resistance_torque_models.h>
+
+#include <deal.II/particles/particle_handler.h>
+
+#include <boost/range/adaptor/map.hpp>
+
+#include <vector>
+
+using namespace dealii;
+using namespace DEM;
+
+#ifndef particles_force_chains_v2_h
+#  define particles_force_chains__v2_h
+
+
+
+template <
+  int                                                       dim,
+  Parameters::Lagrangian::ParticleParticleContactForceModel force_model,
+  Parameters::Lagrangian::RollingResistanceMethod rolling_friction_model>
+class ParticlesForceChains
+  : public ParticleParticleContactForce<dim,
+                                        force_model,
+                                        rolling_friction_model>
+{
+public:
```

Comment:
You do not know it yet so that's not a big deal, but try to add a @brief and a description of all the [in] arguments. You can look at other files to see how this is done :)